root_VS2012/programs/C#/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Views/Ping/Index.cshtml

```diff
@@ -0,0 +1,9 @@
+﻿@{
```

Comment:
@daisukenishino 

Fixed all the [above](https://github.com/OpenTouryoProject/OpenTouryoTemplates/pull/28/files#r49550427) comments.

Please review the pull request
https://github.com/SymphonyTeleca/OpenTouryoTemplates/pull/69